dev-python/opentelemetry-api/opentelemetry-api-1.25.0.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="opentelemetry-python-${PV}"
```

Comment:
Please move that below `inherit`. it doesn't need to be pre-inherit.

(in other files too)